front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -81,7 +82,7 @@ const IdeasNewPage = () => {
         context: {
           type: 'phase',
           action: 'posting_idea',
-          id: phase_id || getCurrentPhase(phases?.data)?.id || '',
```

Comment:
Removing the `getCurrentPhase` fallback and just keeping `phase_id` as I did initially caused a bug when I hit `/ideas/new` directly. Since there was no `phase_id` passed the `Unauthorized` component wouldn't launch the auth modal.